content/articles/javafx-editable-table-view/index.md

```diff
@@ -0,0 +1,442 @@
+Tables are of extreme importance when it comes to data entry and display in applications. Table views support the tabular format display. This kind of format makes it easy to correct and check data patterns.- Ever since the creation of this kind of data display format, and well illustration in Sheet apps e.g. Excel, developers prefer it in TPS and MIS systems.
+
+JavaFX, being focused on UI, has ways to represent data in tabular format. The problem is that one can only view but not edit each cell individually since they aren't editable. This article has solutions to this issue. By the end of the article, the reader will be able to create an editable Table View. That will enable easy data updates directly on the Table being displayed.
+
+### Table of Contents
+
+- [Key takeaways](#key-takeaways)
+- [Pre-requisites](#pre-requisites)
+- [Article structure](#article-structure)
+- [Project initialization](#project-initialization)
+  - [Initial Project Structure](#initial-project-structure)
+  - [New Project Structure](#new-project-structure)
+- [Set up the project, tools, and development environment](#set-up-the-project-tools-and-development-environment)
+- [Table creation and design](#table-creation-and-design)
+- [Add data items on the TableView](#add-data-items-on-the-tableview)
+  - [Define Users model](#define-users-model)
+  - [Setup the controller](#setup-the-controller)
+- [Make the Table cells editable](#make-the-table-cells-editable)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Key takeaways
+
+At the article's tip, the reader will be able to do the following:
+
+- Create a TableView
+- Format the Table
+- Add data items on the TableView
+- Make the TableView editable through the use of its controller
+- Get the edited items
+
+### Pre-requisites
+
+This is an intermediate-level Java user article. For one to easily follow up with the article, the following are the
```

Comment:
Do we have any any beginner level articles to start with?

(I'm also kind of new to this)

Reply:
If so, please add references

Reply:
Let me add one that is at Section.io's blog :+1:

Reply:
Done!